img-versions.json

```diff
@@ -0,0 +1,10 @@
+{
+    "easyengine/mailhog": "v4.0.0-beta.5",
```

Comment:
I think we should be using this docker image version from this file to all the places now. https://github.com/EasyEngine/easyengine/blob/develop-v4/php/EE/Migration/Containers.php#L39

https://github.com/EasyEngine/site-wp-command/blob/develop/src/Site_WP_Docker.php#L128

Reply:
Will updated in https://github.com/EasyEngine/easyengine/blob/develop-v4/php/EE/Migration/Containers.php#L39

For `site-command` and `site-wp-command`:
https://github.com/EasyEngine/site-command/pull/95
https://github.com/EasyEngine/site-wp-command/pull/15